easybuild/easyconfigs/c/Cluster-Buster/Cluster-Buster-0.0-GCC-12.2.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'Cluster-Buster'
+version = '0.0'
```

Comment:
Please use the date of the commit for the version. So `ac1d33cffff0c276216450ebced471595cf01488` is `20200507` (May 7, 2020).